docs/source/conventions.rst

```diff
@@ -0,0 +1,113 @@
+Conventions
+===========
+
+Nouns vs Verbs
+--------------
+
```

Comment:
Please check your reST, you are mixing here and on other place headers, this leads currently to failing tests for me, because your reST is not valid !

:)